src/sib_api_v3_sdk/Client/Configuration.cs

```diff
@@ -183,7 +183,7 @@ public Configuration()
             string tempFolderPath = null,
             string dateTimeFormat = null,
             int timeout = 100000,
-            string userAgent = "sendinblue_clientAPI/v3.0.0/c#"
+            string userAgent = "sendinblue_clientAPI/v#/c#"
```

Comment:
`#` should be replaced by version number, i.e. 3.1.0

Reply:
Its done here:
https://github.com/sendinblue/APIv3-csharp-library/blob/e28fa0fc211adae96102518f9700d5f4037625cd/src/sib_api_v3_sdk/Client/ApiClient.cs#L180